lucene/benchmark/src/java/org/apache/lucene/benchmark/byTask/feeds/DemoHTMLParser.java

```diff
@@ -147,7 +147,7 @@ public InputSource resolveEntity(String publicId, String systemId) {
     }
     
     private static final Set<String> createElementNameSet(String... names) {
-      return Collections.unmodifiableSet(new HashSet<>(Arrays.asList(names)));
+      return Set.copyOf(Arrays.asList(names));
```

Comment:
`Set.of(names)`?

Reply:
actually we should probably remove this method and fix call sites to use `Set.of` instead?